occurrences/admin.py

```diff
@@ -49,6 +49,11 @@ class StudyGroupAdmin(admin.ModelAdmin):
     list_filter = ["created_at", HasUnitIdStudyGroupListFilter]
     search_fields = ["unit_id", "unit_name", "person"]
 
+    def get_form(self, request, obj=None, **kwargs):
+        form = super().get_form(request, obj, **kwargs)
+        form.base_fields["unit_id"].required = False
```

Comment:
This looks like something that would be resolved with havin `blank=True` on the model.

Reply:
But then we go back to our earlier discussion where we talked about whether or not the unit_id should ever be blank 😄 I would say, again, that this is a weakness in Django. The framework should not ever handle a nullable field as a blankable, because null and ´´really are not the same thing and when they are queried, they both needs a check. It's bad for data integrity - Really bad actually, because when a data report is done, you need to be aware of this.

Reply:
okay, of course I could set the required_fields as a custom field, but I would still need to override the get_form -method.

Reply:
Here are only the nullable fields:
```
nullable_fields = [field.name for field in StudyGroup._meta.get_fields() if field.concrete and field.null and not getattr(field, 'blank', None)]
```